frontend-maven-plugin/src/main/java/com/github/eirslett/maven/plugins/frontend/mojo/AbstractFrontendMojo.java

```diff
@@ -59,6 +59,9 @@ public abstract class AbstractFrontendMojo extends AbstractMojo {
     @Parameter(defaultValue = "${repositorySystemSession}", readonly = true)
     private RepositorySystemSession repositorySystemSession;
 
+    @Parameter(property = "useNodeVersionManager", readonly = true, defaultValue = "false")
```

Comment:
I have disabled it by default because there are other node based mojos in the plugin that i didn't touch (most relevant yarn mojo). Could we enable it once we handle yarn as well?

Reply:
I think we might be able to do on by default and off for the BunMojo? All of them require Nodejs other than Bun

Reply:
I have enabled it by default (with exception for bun mojo) now